repro2fhirunittest.py

```diff
@@ -0,0 +1,113 @@
+import unittest
+from reproschema_to_fhir.fhir import add_options, Config, generate_code_system, generate_value_set
+
+class TestReprotoFhir(unittest.TestCase):
+    def testcase_1_add_options(self):
```

Comment:
you shouldn't number the test cases, because if you add a new one you need to increment all the rest, and having a number to refer to it isn't as useful as a very detailed name anyway.